scripts/airgapped/deploy-1.8.sh

```diff
@@ -86,9 +86,20 @@ juju deploy --trust --debug ./$(charm kfp-schedwf) --resource oci-image=$(img sc
 juju deploy --trust --debug ./$(charm kfp-ui) --resource ml-pipeline-ui=$(img frontend)
 juju deploy --trust --debug ./$(charm kfp-viewer) --resource kfp-viewer-image=$(img viewer-crd-controller)
 juju deploy --trust --debug ./$(charm kfp-viz) --resource oci-image=$(img visualization-server)
-juju deploy --trust --debug ./$(charm knative-eventing) --config namespace=knative-eventing
+juju deploy --trust --debug ./$(charm knative-eventing) --config namespace=knative-eventing \
+    --config custom_images="eventing-webhook/eventing-webhook: $(img knative-releases/knative.dev/eventing/cmd/webhook)
+eventing-controller/eventing-controller: $(img knative-releases/knative.dev/eventing/cmd/controller)
+mt-broker-filter/filter: $(img knative-releases/knative.dev/eventing/cmd/broker/filter)
+mt-broker-ingress/ingress: $(img knative-releases/knative.dev/eventing/cmd/broker/ingress)
+mt-broker-controller/mt-broker-controller: $(img knative-releases/knative.dev/eventing/cmd/mtchannel_broker)
+imc-dispatcher/dispatcher: $(img knative-releases/knative.dev/eventing/cmd/in_memory/channel_dispatcher)
+imc-controller/controller: $(img knative-releases/knative.dev/eventing/cmd/in_memory/channel_controller)
+pingsource-mt-adapter/dispatcher: $(img knative-releases/knative.dev/eventing/cmd/mtping)
+migrate: $(img knative-releases/knative.dev/pkg/apiextensions/storageversion/cmd/migrate)
```

Comment:
I think this is part only of knative-serving according to https://github.com/canonical/knative-operators/pull/187/files#diff-6a53f3921c6af1bb033e54860ae6beb7dd00fd7296ddb2c14b1d77e86124e09b.

Reply:
ur right, removed now